### DISCUSSION TITLE
Windows Community Toolkit v8.0 - missing Carousel control in WinUI 3

### DISCUSSION THREAD

**Discussion by adamfierek** [CommunityToolkit/Windows#232] [Feedback]
As in topic.
Is there any substitution?

**Comment by michael-hawker** [comment]
@adamfierek no substitution, we know Carousel is fairly popular but the current implementation is lacking. Rather than invest in porting it we want to invest in creating a modern version which supports more things.

I just transferred a similar question over to Labs, so we'll use that for tracking, see more details there: https://github.com/CommunityToolkit/Labs-Windows/discussions/498
